astropy/io/fits/convenience.py

```diff
@@ -57,10 +57,13 @@
 
 import operator
 import os
+import sys
 import warnings
+import logging
```

Comment:
Ahh, thanks for catching these, got part way through adding logging then stopped.